.env.docker

```diff
@@ -1,2 +1,4 @@
 EMAIL_SMTP_HOST=host.docker.internal
-RABBITMQ_HOST=host.docker.internal
\ No newline at end of file
+RABBITMQ_HOST=host.docker.internal
+ALKEMIO_SERVER_ENDPOINT=http://host.docker.internal/admin/graphql
+SERVICE_ACCOUNT_PASSWORD=rie1@iT0
```

Comment:
this should probably be a value such as 'change-me-now'

Reply:
Removed .env.docker - it's no longer needed. Extracted the defaults in constants, as suggested.